CancerGov/_src/Scripts/NCI/Modules/patternInjector/index.js

```diff
@@ -0,0 +1,57 @@
+import GeoPattern from 'GeoPattern';
+import { getNodeArray } from 'Utilities/domManipulation';
+
+/**
+ * This is a helper library that wraps the GeoPattern external library and simplifies component calls to it for
+ * injecting patterns into the DOM.
+ * 
+ */
+
+const buildPattern = (seed = `${Date.now()}`) => {
+    let pattern;
+    if(typeof seed === 'string') {
+        pattern = GeoPattern.generate(seed);
+    } 
+    else if(typeof seed === 'object') {
+        try {
+            pattern = GeoPattern.generate(seed.str, seed.options)
```

Comment:
I've noticed that we never generate a random seed. Instead the seed is set to an empty string every time in the config files. Could this be written so that when the seed is an object, if `typeof seed.str == undefined` then `seed.str = ''` or use a default varible `GeoPattern.generate(seed.str = '', seed.options)`

Reply:
I'm confused by what you are asking. Right now I do generate a random seed if no string is passed at all.